traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -986,3 +986,36 @@ func CreateDeprecationAlerts(alternative *string) tc.Alerts {
 		return tc.CreateAlerts(tc.WarnLevel, "This endpoint is deprecated, and will be removed in the future")
 	}
 }
+
+// GetLastUpdated checks for the resource in the database, and returns its last_updated timestamp, if available
+func GetLastUpdated(tx *sqlx.Tx, ID int, tableName string) (error, *tc.TimeNoMod) {
```

Comment:
I don't think this is a rule or anything, but generally errors are the last returned value of functions